README.md

```diff
@@ -84,10 +84,10 @@ jobs:
     if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
     steps:
       - uses: foxygoat/howl@v2
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-          SLACK_CHANNEL: C0000000000 # optional; channel ID
-          SLACK_TEXT: <!here> # optional; text or @-mention
-          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
-          DISCORD_TEXT: @here # optional; text or @-mention
+        with:
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
in the action.yaml, these fields have been specified with hyphens - `slack-webhook-url` - and this is pretty standard for github actions (hyphens).

I've just been down a bit of a rabbit hole and there are a number of issues about this because github will set the environment variables `INPUT_SLACK-WEBHOOK-TOKEN` (with hyphens) which are invalid in sh/bash. You can use `"$(printenv INPUT_SLACK-WEBHOOK-TOKEN)"` to access and since its all in one place at the top, it's not too bad, but it is still a little sour.

Reply:
hooray for 🐰 🕳️ 
all addressed i think. thank you!